benchmarks/twMerge.bench.ts

```diff
@@ -0,0 +1,37 @@
+import { bench, describe } from 'vitest'
+
+import { createTailwindMerge, getDefaultConfig, twMerge as twMergeCached } from '../src'
+
+import testData from './perf-test-data.json'
+
+for (let [twMerge, cached] of [
+    [twMergeCached, true],
+    [createTailwindMerge(() => ({ ...getDefaultConfig(), cacheSize: 0 })), false],
```

Comment:
> I feel that I need to add benchmarks without cache. Because now we are benchmarking LRUCache performance. It's ok, but I do not think that's what we want

Yeah, the cache is an issue for benchmarks. Especially if the benchmark runs multiple times, then we're really only testing the cache which doesn't make much sense. [In the Vitest docs it says that each bench is run 5 times as a warmup](https://vitest.dev/api/#bench:~:text=/**%0A%20%20%20*%20warmup%20iterations%0A%20%20%20*%20%40default%205%0A%20%20%20*/%0A%20%20warmupIterations%3F%3A%20number), so that's a problem.

But testing without the cache also doesn't make much sense in the collection bench since you'll hit the cache quite often when using `twMerge` in a real app, which the collection bench tries to mimic.

Moreover, there is the problem that tailwind-merge initializes its internal data structures lazily on the first `twMerge()` call, adding a hefty performance penalty to the first bench run.

We can eliminate all of these issues by create a new `twMerge` for every bench run and calling it once before the bench so that the cache is empty and the internal data structures are initialized before every run.

I think we can do it with the `setup` function which is called before every bench run (I hope):

```ts
let twMerge

function setup() {
    // Creates a new `twMerge` instance with an empty cache
    twMerge = extendTailwindMerge({ cacheSize: cached ? undefined : 0 })
    // Sets up internal data structures
    twMerge('')
}

bench(/* bench name */, /* bench function */, { setup })
```